sys/include/libc.h

```diff
@@ -599,6 +599,7 @@ extern	int	chdir(const char*);
 extern	int	close(int);
 extern	int	create(const char*, int, uint32_t);
 extern	int	dup(int, int);
+extern  char*   errno2str(uint errno);
```

Comment:
I know we don't tend to make a big fuss about formatting but...

Reply:
tell GitHub to stop displaying this incorrectly. It lines up right if you look at the raw file.

Reply:
anyway, fixed, I screwed up and did not use tabs. My bad.